migrations/2018_01_15_071700_change_user_discussions_discussions_users.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/*
+ * This file is part of Flarum.
+ *
+ * (c) Toby Zerner <toby.zerner@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use Illuminate\Database\Schema\Builder;
+
+return [
+    'up' => function (Builder $schema) {
+        $schema->rename('users_discussions', 'discussions_users');
+    },
+
+    'down' => function (Builder $schema) {
+        $schema->rename('discussions_users', 'users_discussions');
+    }
+];
```

Comment:
This could use the `Migration::renameTable`` helper?
Also for consistency the migration should be called `rename_users_discussions_to_discussions_users.php`